GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/Domains/1301_en-gb.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Domain Key="b1d5f521-0000-0000-0000-000000000000" Alias="*1301">
+  <Info>
+    <IsWildcard>true</IsWildcard>
+    <Language>en-GB</Language>
+    <Root Key="a8cfa099-8165-4cfe-b3d1-179472290d0c">/Home/Textarea</Root>
```

Comment:
It's not usual to define 'hostnames and cultures' on each page. You can remove it from these 3 pages and instead define it on the home page, which is where English and German are set up.